apps/prairielearn/src/lib/groups.sql

```diff
@@ -169,7 +169,8 @@ FROM
   JOIN group_users AS gu ON gu.group_id = g.id
 WHERE
   gc.assessment_id = $assessment_id
-  AND gu.user_id = $user_id;
+  AND gu.user_id = $user_id
+  AND g.deleted_at IS NULL;
```

Comment:
Groups are soft-deleted; I believe we want to disregard membership in deleted groups here.